### PR TITLE
NEXUS-10061 - Integration API return values

### PR DESCRIPTION
**Pull request by kellyrob99**
couple of examples for how to return values from scripts

https://issues.sonatype.org/browse/NEXUS-10061

### CODE REVIEW

**Comment by mrprescott** [comment]
+1

File: scripting/complex-script/npmAndBowerRepositories.groovy [hunk 1]
```diff
@@ -1,15 +1,38 @@
-repository.createNpmProxy('npmjs-org', 'https://registry.npmjs.org')
+import groovy.xml.MarkupBuilder
+def newRepositories = []
+newRepositories << repository.createNpmProxy('npmjs-org2', 'https://registry.npmjs.org')
 
-repository.createNpmHosted('npm-internal')
+newRepositories << repository.createNpmHosted('npm-internal2')
```
Comment (mosabua) [inline scripting/complex-script/npmAndBowerRepositories.groovy]: Why the 2? The names are the same as they are in the documentation itself on purpose...

Comment (kellyrob99) [inline scripting/complex-script/npmAndBowerRepositories.groovy]: damn, sorry - local testing change slipped through, will fix

**Comment by joedragons** [comment]
+1, these all run as expected after fixes.  Few errors I saw are expected with baseurl running thru the console.  Thanks!  (PS: Will re-review the chapter _sheepish_)

**Comment by kellyrob99** [comment]
Merged after getting @joedragons to validate the changes

**Comment by mosabua** [comment]
Nice work guys!

**Pull request merged**